tasks/image/tag.go

```diff
@@ -24,6 +25,14 @@ func tagImage(ctx *context.ExecuteContext, t *Task, imageTag string) error {
 		return nil
 	}
 
+	switch strings.Count(imageTag, ":") {
+	case 1:
+	case 2:
+		imageTag = imageTag[strings.Index(imageTag, ":")+1:]
+	default:
+		return fmt.Errorf("Not a valid tag")
```

Comment:
needs case 1 to figure out whenever its a repo or a tag depending on the existence of a slash